src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,82 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { invoke } from 'lodash';
+// Relative imports.
+import { capitalize } from '../../helpers';
+
+const SearchResult = ({ school }) => (
+  <div
+    className={classnames(
+      'medium-screen:vads-l-col',
+      'vads-l-row',
+      'vads-u-background-color--gray-light-alt',
+      'vads-u-margin-bottom--1',
+      'vads-u-margin-right--1',
+      'vads-u-padding--1',
+    )}
+  >
+    <div className={classnames('vads-l-col--8')}>
+      {/* School Name */}
+      <h3 className={classnames('vads-u-margin--0')}>
```

Comment:
Love this. My plan was actually to remove the `classnames` that don't have any conditions in them, but added them initially in case there were conditions I would add to them later on. Before I pull this PR out of draft mode, I'll be sure to clean them up! 🙂